bower.json

```diff
@@ -25,13 +25,13 @@
     "marked-element": "PolymerElements/marked-element#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
     "paper-button": "PolymerElements/paper-button#^1.0.0",
-    "polymer": "Polymer/polymer#^1.0.0",
+    "polymer": "Polymer/polymer#^1.2.3",
```

Comment:
Not needed. I've just gotten  in the habit of keeping deps up to date. I can remove.